com.unity.netcode.gameobjects/Documentation~/components/core/networkobject.md

```diff
@@ -46,6 +46,9 @@ To spawn NetworkObjects with ownership use the following:
 ```csharp
 GetComponent<NetworkObject>().SpawnWithOwnership(clientId);
 ```
+> [!NOTE]
+> When using the `SpawnWithOwnership` method, be aware that any component that has owner-specific checks to perform specific actions won't be invoked on the spawn authority  side during the spawn sequence. The spawn authority is the server when using a client-server network topology, and can be any client when using a distributed authority network topology. Using `SpawnWithOwnership` can impact things like [NetworkTransform](../helper/networktransform.md) when using an owner authority motion model, and potentially provide undesired parenting artifacts and/or impact your own scripts if you are planning to have the spawn authority make any further post-spawn adjustments within the same frame.
```

Comment:
If we link the ownership overview above, I think we can simplify this wording some.

```suggestion
> The `SpawnWithOwnership` method allows a game client to spawn an object that it doesn't own. This means any owner-specific checks during the spawn sequence will not be invoked on the game client who invoked the spawn. Using `SpawnWithOwnership` can result in unexpected behaviour if the game client who invokes the spawn (the spawn authority) makes any further post-spawn adjustments within the same frame.
```

(though looking at what's in that ownership page, I should expand the info about the motion model authority on that page)